docs/configuration/environment-variables.md

```diff
@@ -22,7 +22,7 @@ env:
 
 Kamal uses dotenv to automatically load environment variables set in the `.kamal/secrets` file.
 
-If you are using destinations, secrets will instead be read from `.kamal/secrets-<DESTINATION>` if it exists.
+If you are using destinations, secrets will instead be read from `.kamal/secrets.<DESTINATION>` if it exists.
 
 Common secrets across all destinations can be set in `.kamal/secrets-common`.
```

Comment:
Maybe after all `secrets-<DESTINATION>` would more sense, in order to be consistent with `secrets-common`?

Reply:
See also here: https://github.com/basecamp/kamal/pull/933#pullrequestreview-2297272674

Reply:
Thanks @visini! The docs are indeed wrong, `secrets-common` uses a dash so that you could have a destination called `common`.